services/reading_mail/reading_mail.js

```diff
@@ -0,0 +1,22 @@
+const superagent = require('superagent')
+const mailsac_api_key = 'YOUR_API_KEY_HERE'
+
+superagent
+  .get('https://mailsac.com/api/addresses/calendartrinity@mailsac.com/messages')
+  .set('Mailsac-Key', mailsac_api_key)
+  .then((messages) => {
+      messageId = messages.body[0]._id
```

Comment:
might want a `const` there to avoid hoisting to global scope if somebody copy+pastes this